_data/accessibility-tests/collection.yml

```diff
@@ -0,0 +1,85 @@
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 1.3.1
+  - summary: Collection works on mobile devices.  
+    summary_additional: When you use a mobile device or tablet, collection content is visible in both portrait and landscape views with no loss of functionality.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 1.3.4
+  - summary: Color is not the only method used to convey meaning.   
+    summary_additional: When you view collection content, links, tags and other text all help establish context.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 1.4.1
+  - summary: Links, text and tags in the collection meet color contrast requirements.   
+    summary_additional: When you use a color contrast checker to view a collection, contrast between links, text and tag colors against the background is at least 4.5:1.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 1.4.3
+  - summary: All headings are clear and descriptive.   
+    summary_additional: When you view the collection, the headings are clear and concise.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 2.4.6
+# Keyboard navigation tests
+  - summary: Collection content is accessible using only keyboard. 
+    summary_additional: When you use a keyboard, you can navigate to links, buttons and content, and navigate away.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.2
+    wcag_criterion: 2.1.1
+  - summary: Collection content does not trap keyboard focus.  
+    summary_additional: When you tab through collection content with a keyboard, you can easily move in and out of collection elements to get to other content.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.2
+    wcag_criterion: 2.1.2
+  - summary: Focus follows logical content order. 
+    summary_additional: When you tab through a collection with a keyboard, focus on links and other interactive elements stays in logical order in relation to the other content.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.2
+    wcag_criterion: 2.4.3
+  - summary: Focus indicator is clearly visible. 
+    summary_additional: When you use a keyboard to focus on a collection, you will see a visible outline or other clear indication of focus for interactive elements.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.2
+    wcag_criterion: 2.4.7
+# Screen reader tests
+  - summary: Screen reader only announces icons or graphics marked with alt text.
+    summary_additional: When you open the collection with screen reader, you don't hear any information about icons or images marked as decorative. 
+    test_status: conditional
+    test_type: screen_reader
+    version_tested: 3.8.2
+    wcag_criterion: 1.1.1
```

Comment:
**Suggestion:** Update the description to match [icon's 1.1.1 summary](https://arc.net/l/quote/riqvddqr)

> "**Icons provide a text alternative for screen readers when needed.** When you use a screen reader and navigate to a non-decorative icon, you hear an appropriate description of the icon. If the icon is purely decorative, you should not hear a description."

```suggestion
  - summary: Icons or graphics provide a text alternative for screen readers when needed.    
    summary_additional: When you use a screen reader and navigate to a non-decorative icon or graphic, you hear an appropriate description of the icon. If the icon is purely decorative, you should not hear a description.
    test_status: conditional
    test_type: screen_reader
    version_tested: 3.8.2
    wcag_criterion: 1.1.1
```